fortran/src/H5_ff.F90

```diff
@@ -701,7 +701,7 @@ END SUBROUTINE h5close_f
 !
 ! Outputs:
 !  majnum - major version of the library
-!  minum  - minor version of the library
```

Comment:
The correct spelling should be minnum - the actual variable name.

Reply:
Yes. This needs to be reverted.

Reply:
This could probably be avoided if we name the variables maj_num and min_num, but I don't want to have to engineer around a bone-headed spelling checker that wants to rename variables.

Reply:
This is a comment, how is the spell check supposed to know it is a variable? This was reverted already.